internal/settings/settings.go

```diff
@@ -290,6 +290,24 @@ type ExternalConfig struct {
 	} `json:"gen,omitempty" yaml:"gen,omitempty"`
 }
 
+// Validate returns an error if any of the restricted, yet previously supported,
+// ExternalConfig attributes are set. This includes:
+//   - NoDefaultExcludes
+//   - Lint.Group
+//   - Gen.GoOptions.NoDefaultModifiers
+func (e ExternalConfig) Validate() error {
+	if e.NoDefaultExcludes {
```

Comment:
Actually, we're going to want to make this the new default, and make this true, or alternatively remove anything from the list of default excludes in https://github.com/uber/prototool/blob/dev/internal/settings/settings.go#L49

Reply:
Ah yea good point. Nice catch, I'll update.